specs/shell/make_a_dir_and_change_into.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: Make a directory and change into it
+command: mkdir {{dirname}} && cd $_
```

Comment:
nit: we will need to specify the shells that this can run on. I believe `$_` is not valid in fish.

Reply:
Yeah, nice catch!

I wonder if we can unify the syntax across shells using some history indexing tricks `!:-1` or smth, but it doesn't seem to work universally either.

ok I'll add `bash` `zsh` fro now